Bottom: 8327c888e3f0457f27a613569ea59c26315eb645
Top:    479a7e9cd93dcf42671b02a5fec98ef7420242d1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-30 10:45:15 +0000

Refresh of fix-auto-enforce-drc-with-spli

---

diff --git a/src/set.c b/src/set.c
index 9fcc4dd..324e625 100644
--- a/src/set.c
+++ b/src/set.c
@@ -300,7 +300,7 @@ SetMode (int Mode)
       Crosshair.AttachedLine.State = STATE_FIRST;
       if (Mode == LINE_MODE && TEST_FLAG (AUTODRCFLAG, PCB))
 	{
-	  if (ClearFlagOnAllObjects (true, FOUNDFLAG))
+	  if (ClearFlagOnAllObjects (true, CONNECTEDFLAG | FOUNDFLAG))
 	    {
 	      IncrementUndoSerialNumber ();
 	      Draw ();
